src/cryptonote_config.h

```diff
@@ -207,6 +207,16 @@ namespace config
   std::string const GENESIS_TX = "013c01ff0001ffffffffffff03029b2e4c0281c0b02e7c53291a94d1d0cbff8883f8024f5142ee494ffbbd08807121017767aafcde9be00dcfd098715ebcf7f410daebc582fda69d24a28e9d0bc890d1";
   uint32_t const GENESIS_NONCE = 10000;
 
+  // Hash domain separators
+  const std::string HASH_KEY_BULLETPROOF_EXPONENT("bulletproof");
```

Comment:
Why is this a `std::string`? It requires initialization at runtime, and the order of this initialization is undefined. A string can be constructed when its needed, it looks like temporary string construction is used anyway.

Reply:
It was taken from the corresponding bulletproofs code.

Reply:
Updated.